SEQLinkage/Runner.py

```diff
@@ -51,7 +53,8 @@ def format_plink(tped, tfam):
         with open(out_base + '.ped', 'w') as p:
             for line in tfam_fh:
                 p.write(line.strip())
-                list(map(lambda x: p.write(' {} {}'.format(x.popleft(), x.popleft)), geno))
+                s = line.strip().split()
```

Comment:
do u use this `s` later? if not please remove

Reply:
Yes. without this line, the format will be wrong.